From 2e51fb9163bbc6b9a27fa524382c5aad9700dd0d Mon Sep 17 00:00:00 2001
From: Miklos Vajna <vmiklos@collabora.co.uk>
Date: Tue, 8 Aug 2017 17:23:38 +0200
Subject: [PATCH 2/3] EPUBGenerator: don't write empty guide element

<http://www.idpf.org/epub/20/spec/OPF_2.0.1_draft.htm#Section2.6> "2.6:
Guide" requires:

	Within the package there may be one guide element, containing one or
	more reference elements.

Given that we never wrote reference child-elements, just don't write the
guide element at all, especially seeing that it's deprecated in OPF3.
---
 src/lib/EPUBGenerator.cpp | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/src/lib/EPUBGenerator.cpp b/src/lib/EPUBGenerator.cpp
index 507eb14..9dbccac 100644
--- a/src/lib/EPUBGenerator.cpp
+++ b/src/lib/EPUBGenerator.cpp
@@ -225,9 +225,6 @@ void EPUBGenerator::writeRoot()
   m_htmlManager.writeSpineTo(sink);
   sink.closeElement("spine");
 
-  sink.openElement("guide");
-  sink.closeElement("guide");
-
   sink.closeElement("package");
 
   sink.writeTo(*m_package, "OEBPS/content.opf");
-- 
2.12.3

